app/components/toggle.ts

```diff
@@ -6,13 +6,28 @@ interface Signature {
   Args: {
     isDisabled?: boolean;
     isOn: boolean;
+    isVisible?: boolean;
```

Comment:
Where is `isVisible` used?

Reply:
In general it's a bad idea to accept `isVisible` for any component - a caller should just not render the component, or it should use styling to override (`class="hidden"` or `class="opacity-0"`)